js_modules/dagit/packages/core/src/runs/RunsFilterInputNew.tsx

```diff
@@ -157,12 +162,7 @@ interface RunsFilterInputProps {
 }
 
 // Exclude these tags from the "tag" filter because theyre already being fetched by other filters.
-const tagsToExclude = [
-  DagsterTag.User,
-  DagsterTag.ScheduleName,
-  DagsterTag.SensorName,
-  DagsterTag.Backfill,
-];
+const tagsToExclude = [...CREATED_BY_TAGS, DagsterTag.Backfill];
```

Comment:
Ahh nice this is good cleanup 👍  I wonder if these should actually be part of the file with DagsterTag? Might be good to try grepping for `DagsterTag.SensorName` and see if there are other places we're excluding this same set at the component level

Reply:
That's a good point. I think as we expand filtering to other screens we'll probably end up doing that.